### ISSUE TITLE
about extclsshpinfo.py

### ISSUE THREAD

**Issue by anruoxi123** [wentaozhu/DeepLung#77]
Hello, teacher.

When I run extclsshpinfo.py, the following problems occur,

I would like to ask whether the emergence of this question is related to the mapping of LIDC dataset and LUNA16 dataset.

I spent a long time looking for the cause of the problem, but I couldn't find it.

I hope you can give me some advice.
![5ff1ade3bdf27d999112624d69a217f](https://user-images.githubusercontent.com/43407427/59682699-516d7680-9209-11e9-843b-09eaa2ff8846.png)

**Comment by wentaozhu** [comment]
I think it is easy to debug. Check what the file is and check whether the id is in the patient.

**Comment by anruoxi123** [comment]
Hello, I have checked the file, which is available in the files of lidc-idri-mappingluna16 and list3.2.csv.

But annotations are also available in the annotation.csv file,

I'm trying to remove that tag in the annotation.csv file,

But then there's another labeling error.

I'm a little confused about whether these tags should be deleted,

After I replace annotationdetclsconvfnl_v3.csv in the extclsshpinfo.py file with the annotationdetclsconvfnl_v3.csv that comes with the program, the program can execute, and it generates a bunch of files,

But when I executed the humanperformance-py file, I couldn't find many files,

May I ask what these documents are? Does it have to be self-generated?
![45a28c8624a4b2bf037884e7cbc9c79](https://user-images.githubusercontent.com/43407427/59748342-7f0bfb80-92ad-11e9-941d-dafde3fb1bca.png)
![52b78067b4901cce46e412983cccb78](https://user-images.githubusercontent.com/43407427/59748345-829f8280-92ad-11e9-8c22-ba1b6158875e.png)

**Comment by wentaozhu** [comment]
What is the name of the file?

**Comment by anruoxi123** [comment]
It is extclsshpinfo.py.
And I've now removed the tags that report errors to display from the annotation.csv file,

There were 1186 lines of tags before, and I deleted more than 500 lines.

But now there's another problem, which is line 184 of the extclsshpinfo.py file,

It says the file doesn't exist, I checked the file, it doesn't exist,

It extracts the name of the LUNA16 data set,

But what is the path of the LIDC?
![4a4fc7e3d1b7e67e407d6c062f38a7c](https://user-images.githubusercontent.com/43407427/59816473-cb5b4800-934e-11e9-9b9b-db8cbf7e7a74.png)
![e16ab1a1a233d5a723c500e74a87e45](https://user-images.githubusercontent.com/43407427/59816474-cbf3de80-934e-11e9-8f0f-82bdeffbdba4.png)
